NameGame.php

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Name Game</title>
+	</head>
+	
+	<body>
+		<h1>Please enter you name below...</h1>
+			<form action="index.php" method="get">
```

Comment:
spacing here, the <form> and sub tags should be on the same spacing level as the <h1> because they are the same element depth.